breathecode/admissions/tests/urls/tests_academy_cohort.py

```diff
@@ -1405,6 +1405,39 @@ def test_academy_cohort_delete_without_args_in_url_or_bulk(self):
         }])
         self.assertEqual(self.all_cohort_time_slot_dict(), [])
 
+    @patch(GOOGLE_CLOUD_PATH['client'], apply_google_cloud_client_mock())
+    @patch(GOOGLE_CLOUD_PATH['bucket'], apply_google_cloud_bucket_mock())
+    @patch(GOOGLE_CLOUD_PATH['blob'], apply_google_cloud_blob_mock())
+    def test_academy_cohort_delete_in_bulk_with_students(self):
+        """Test /cohort/:id/user without auth"""
+        self.headers(academy=1)
+        many_fields=['id']
+
+        base = self.generate_models(academy=True, capability='crud_cohort', role='potato')
+
+        expected = {
+            'detail': 'cohort-has-students',
+            'status_code': 400,
+        }
+
+        for field in many_fields:
+            model = self.generate_models(authenticate=True, profile_academy=True, 
+                cohort_user=True, models=base)
+            
+            value = getattr(model['cohort'], field)
+
+            url = (reverse_lazy('admissions:academy_cohort') + f'?{field}=' +
+                str(value))
+            response = self.client.delete(url)
+            json = response.json()
+
+            self.assertEqual(json, expected)
+            self.assertEqual(self.all_cohort_dict(), [{
+                **self.model_to_dict(model, 'cohort')
+            }])
+
+        self.assertEqual(response.status_code, status.HTTP_400_BAD_REQUEST)
```

Comment:
This line should be moved to the 1435, actually just is tested the last iteration